.circleci/config.yml

```diff
@@ -0,0 +1,82 @@
+# Use the latest 2.1 version of CircleCI pipeline process engine. See: https://circleci.com/docs/2.0/configuration-reference
+version: 2.1
+executors:
+    main-env:
+        docker:
+          - image: centos:centos7.6.1810
+        environment:
+            JVM_MEMORY: 2200m # Customize the JVM maximum heap limit
```

Comment:
The JVM has a container mode that might play nicer than having to explicitly set something but if this works, it works.

Reply:
added to issues